.github/workflows/ci.yml

```diff
@@ -0,0 +1,63 @@
+name: GitHub CI
+
+on: [push, pull_request]
+
+jobs:
+  ubuntu-build-test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install deps
+      run: |
+        sudo apt-get update -y
+        sudo apt-get install -y libevent-dev libseccomp-dev git libsasl2-dev
```

Comment:
doesn't need libio-socket-ssl-perl or libssl-dev?

also ubuntu-latest is always the LTS?

Reply:
> doesn't need libio-socket-ssl-perl or libssl-dev?

GItHub environent already installed this by default.

> also ubuntu-latest is always the LTS?

Supposedly yes. As of this time, it's still 18.04 even though 20.04 already released. Don't know when GitHub will use the 20.04 version.

Reply:
Usually the minimum time is after the 20.04.1 version releases a month after the main release.